recipes/lora_dpo_single_device.py

```diff
@@ -180,11 +187,13 @@ def setup(self, cfg: DictConfig) -> None:
         # log config with parameter override
         self._metric_logger.log_config(cfg)
 
+        self._model_compile = cfg.compile
```

Comment:
nit: `self._is_model_compiled` or similar is a bit more clear

Reply:
fyi this is what we use in all our recipes - can amend